gateway-messages/src/lib.rs

```diff
@@ -0,0 +1,149 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+pub mod sp_impl;
+
+use bitflags::bitflags;
+use serde::{Deserialize, Serialize};
+
+pub use hubpack::error::Error as HubpackError;
+pub use hubpack::{deserialize, serialize, SerializedSize};
+
+pub mod version {
+    pub const V1: u32 = 1;
```

Comment:
for my own education: why not use an enum with a value?

Reply:
serde doesn't have a way to serialize enum values as their integer representations by default. I could [pull in serde_repr](https://serde.rs/enum-number.html), though. I'm ambivalent about this - thoughts?

Reply:
👍